instance/github.py

```diff
@@ -133,6 +133,18 @@ def get_team_from_organization(organization_name, team_name='Owners'):
     raise KeyError(team_name)
 
 
+def get_watched_usernames():
+    """
+    Retrieve the usernames to watch from both the watched organization and
+    username settings.
+    """
+    usernames = []
+    if settings.WATCH_ORGANIZATION:
+        usernames = get_username_list_from_team(settings.WATCH_ORGANIZATION)
```

Comment:
Can you append the existing list too here? Just to be consistent, so that someone doesn't miss that the list is overridden if something is added before this step.